db/migrate/20230503183515_add_post_ref_to_likes.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Add a reference to the users table to the likes table.
+class AddPostRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :post, index: true, foreign_key: { to_table: :posts }
+  end
+end
```

Comment:
- [ ] Since you have added this migration, 🤔  you no longer need to implement the previous requirement but you have to delete this column ` t.integer :postId` from your CreateLikes migrations.